source/collection-management/tests/modifyCollection-errorResponse.yml

```diff
@@ -0,0 +1,59 @@
+description: "modifyCollection-errorResponse"
```

Comment:
For the benefit of test coverage, have you considered implementing the operation via `runCommand` in your test suite? PHP has a helper for it but we don't actually model all of the possible parameters, so it should be quite trivial to just build a command document from the `arguments` in the test operation if you wanted.

Maybe not so relevant for this case, since the purpose of the test is to ensure a _helper_ provides appropriate error access, but it could allow you to run some other tests that happen to use `modifyCollection`.

Reply:
Yeah an interesting point; I imagine users are already running `collMod` through our `RunCommand` API, so would be good to test it. Filed [GODRIVER-2587](https://jira.mongodb.org/browse/GODRIVER-2587).